In [ ]:
import pandas as pd
import numpy as np
import scipy.stats as stats

In [ ]:
x=45
std=40
mean=50

In [ ]:
# t-value = (x-mean)/(sample_std/n**0.5)
t = np.round(stats.t.ppf(0.025, df = 249),2)
t

-1.97

In [ ]:
# n = (sample_standard_deviation*tscore)/(sample_mean=population_mean)
n = ((std*abs(t)) / (5))**2
n

248.3776

In [ ]:
df= n-1
print(n,df)

248.3776 247.3776


hence,if the auditors would like to maintain the probability of investigation to 5% then they should sample 248 transactions.